config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
```

Comment:
Don't know how to be clearer that `NodeMetricsPort` to say "this is the port that metrics from the node are collected from" :)